scripts/functions.js

```diff
@@ -1,612 +1,65 @@
-// store this as a global variable so that the stats box can always access the current value
-var filterStates = {
-	year: false,
-	district: false
-};
-// store the year relating to any currently-displayed popup, so it can be cleaned up if necessary
-var popupYear = 0;
-// assign all new popups to this variable so we can remove them as needed
-var popup;
-// first we check for the URL parameter '?districts=senate'.  If found, we'll show state senate districts; otherwise state house
-var urlParams = {};
-window.location.href.replace(
```

Comment:
There's a lot of pruning of parent project code in this PR.  It would have been tidier to keep it separate, but I ended up using some of the old functions as a reference for how to do things here, so now I know I can safely get rid of them.